tests/bundle/test_bundle_adjustment.py

```diff
@@ -33,14 +34,7 @@ def test_simple_scene(self):
 
         expected_error = 0.046137573704557046
 
-        self.assertTrue(
-            np.isclose(
-                expected_error,
-                computed_result.total_reproj_error,
-                atol=1e-2,
-                rtol=1e-2,
-            )
-        )
+        self.assertTrue(np.isclose(expected_error, computed_result.total_reproj_error, atol=1e-2, rtol=1e-2))
```

Comment:
do we need both atol and rtol? I prefer just atol

Reply:
Larger scenes might have larger absolute errors?

Reply:
maybe. i dont know about that, though